docs/registry/create/_index.md

```diff
@@ -58,7 +58,20 @@ This interface defines how your model's server responds to API requests.
 
 To ensure the client interface you create returns the expected results, use the appropriate client interface defined in <file>components/\<resource-name\>/client.go</file> or <file>services/\<resource-name\>/client.go</file> in the [Viam RDK](https://github.com/viamrobotics/rdk/blob/main/) as a reference.
 
-For example, the `base` component client is defined in the [<file>client.go</file>](https://github.com/viamrobotics/rdk/blob/main/components/base/client.go) file.
+For example, the `base` component client is defined in the [<file>components/base/client.go</file>](https://github.com/viamrobotics/rdk/blob/main/components/base/client.go) file.
+
+See [Valid APIs to implement in your model](#valid-apis-to-implement-in-your-model) for more information.
+
+{{% /tab %}}
+{{% tab name="C++" %}}
+
+To create a new resource model, you need to implement your model's **client** interface in a file called `my_modular_resource.cpp`.
```

Comment:
I too may be misunderstanding! I want to try to leave initial mention at `cpp` because its wholly possible to write a module using [just one `cpp` file](https://github.com/viamrobotics/viam-cpp-sdk/tree/main/src/viam/examples/modules/simple).

But certainly many uses will also include `hpp` files, so I've added a new sentence on line 76-77 below to mention. LMK if I'm close!